cmd/globaldns.go

```diff
@@ -0,0 +1,931 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/rancher/cli/cliclient"
+	managementClient "github.com/rancher/types/client/management/v3"
+	"github.com/urfave/cli"
+)
+
+const (
+	alidnsProvider     = "alidns"
+	cloudflareProvider = "cloudflare"
+	route53Provider    = "route53"
+
+	envAlibabaCloudAccessKey = "ALI_ACCESS_KEY_ID"
+	envAlibabaCloudSecretKey = "ALI_ACCESS_KEY_SECRET"
+	envAwsAccessKey          = "AWS_ACCESS_KEY_ID"
```

Comment:
Make these match the UI  e.g. `AWS_ACCESS_KEY`

Reply:
Where can I see `AWS_ACCESS_KEY` in the UI? I use this because it is commonly used by aws cli/sdk

Reply:
That makes sense, if that is how the aws cli works leave it.